misc/random.c

```diff
@@ -51,6 +51,8 @@ struct mp_rand_state mp_rand_seed(uint64_t seed)
             return ret;
         }
         seed = mp_raw_time_ns();
+        seed ^= (uintptr_t)&mp_rand_seed; // ASLR, hopefully
```

Comment:
I don't see the value in these hacks. If av_get_random_bytes fails then we already are in a relatively broken environment.

Reply:
They are cheap, so might as well have it, though I agree it's bit of hackery for not much gain. (also on 32-bit build it will init only lower bits, but I guess that's fine, because splitmix will distribute them. hopefully).